static/app/types/index.tsx

```diff
@@ -1659,7 +1659,7 @@ export type NewQuery = {
   end?: string;
 
   // Graph
-  yAxis?: string;
+  yAxis?: string | string[];
```

Comment:
This kind of mixed types could get confusing and hard to maintain? Should we just switch to implementing it as a list of string everywhere to keep it consistent?

Reply:
👍  I've updated yAxis to just `string[]`, there were a few dependant areas I needed to update to support this but nothing major.